plugins/org.jkiss.dbeaver.data.gis.view/src/org/jkiss/dbeaver/ui/gis/panel/GISLeafletViewer.java

```diff
@@ -117,13 +117,15 @@ public GISLeafletViewer(Composite parent, @NotNull DBDAttributeBinding[] binding
         try {
             browser = new Browser(composite, SWT.NONE);
         } catch (SWTError error) {
-            log.error("Internal web browser initialization failed", error);
-            for (Control control : composite.getChildren()) {
-                control.dispose();
-            }
+            log.warn("Internal web browser initialization failed", error);
             browser = null;
             if (error.code != SWT.ERROR_NOT_IMPLEMENTED) {
+                for (Control control : composite.getChildren()) {
+                    control.dispose();
+                }
                 throw error;
+            } else {
+                browser = new Browser(composite, SWT.WEBKIT);
```

Comment:
Why `SWT.WEBKIT`?

Reply:
Little hack, Eclipse doesn't SWT.IE(which is odd) but if it tries to create Webkit it will use IE instead